package/yast2-squid.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Tue Dec 11 08:35:22 UTC 2018 - jreidinger@suse.com
+
+- always use absolute path to binaries (bsc#1118291)
+- properly escape shell arguments (bsc#1118291)
```

Comment:
Actually, what I see here is using Ruby to change the file permissions instead executing a shell command, ins't it?

Reply:
yes, it is just copy paste of previous changes. But in fact it say the goal of change. Prevent shell injection and using PATH dependent path to binaries, which I fixed.